src/components/Home/Home.jsx

```diff
@@ -29,6 +43,7 @@ const Home = ({ users }) => {
     <div className="home-container">
       <h1>Home</h1>
       <Search setSearchValue={setSearchValue} />
+      <SkillsFilter setFilterValue={setFilterValue} />
```

Comment:
Try and group the search bar and the filters into a single div and give it a class name of search-container. This willl make styling with css much easier.